float64 bearing_angle
---
